rust/tw_aptos/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "tw_aptos"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+tw_coin_entry = { path = "../tw_coin_entry" }
+tw_keypair = { path = "../tw_keypair" }
+tw_proto = { path = "../tw_proto" }
+tw_number = { path = "../tw_number" }
+tw_hash = { path = "../tw_hash" }
+tw_memory = { path = "../tw_memory" }
+bcs = "0.1.6"
+move-core-types = { git = "https://github.com/move-language/move", rev = "ea70797099baea64f05194a918cebd69ed02b285", features = ["address32"] }
```

Comment:
We already use `move-core-types`, but `f7137eabc2046f76fdad3ded2c51e03a3b1fbd01` revision. Is it possible to use the same revision in `tw_move_parser` and `tw_aptos`?